### PR TITLE
removed the usages of sys.exit and print

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/demisto-sdk/pull/718

## Description
removed the usages of sys.exit and print

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 1]
```diff
@@ -28,7 +28,6 @@ def handle_error(message, is_return_error):
         return_error(message)
     else:
         demisto.results(message)
-        sys.exit(1)
```
Comment (bakatzir) [inline Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:31]: i think sys.exit(0) is needed here as we do not want to continue.

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml [hunk 2]
```diff
@@ -125,6 +125,6 @@ tags:
 - phishing
 timeout: '0'
 type: python
-dockerimage: demisto/dl:1.3
+dockerimage: demisto/dl:languages1.0
```
Comment (bakatzir) [inline Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml:128]: @lperryp / @erezh31 is this ok?

File: Packs/CortexXDR/ReleaseNotes/2_3_4.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Converted the usage of print function wi
```
Comment (bakatzir) [inline Packs/CortexXDR/ReleaseNotes/2_3_4.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

File: Packs/CommonScripts/ReleaseNotes/1_2_36.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### JSONtoCSV
+- Updated the use of print function with debug function.
```
Comment (bakatzir) [inline Packs/CommonScripts/ReleaseNotes/1_2_36.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

File: Packs/CrowdStrikeFalconX/ReleaseNotes/1_0_2.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CrowdStrike Falcon X
+- Converted the usage of print function with debug function.
```
Comment (bakatzir) [inline Packs/CrowdStrikeFalconX/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

File: Packs/SafeBreach/ReleaseNotes/1_0_6.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### SafeBreach v2
+- Converted the usage of print function with debug function.
```
Comment (bakatzir) [inline Packs/SafeBreach/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

File: Packs/URLHaus/ReleaseNotes/1_0_1.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### URLhaus
+- Converted the usage of print function with debug function.
```
Comment (bakatzir) [inline Packs/URLHaus/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

File: Packs/Whois/ReleaseNotes/1_1_5.md [hunk 8]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Whois
+- Converted the usage of print function with debug function.
```
Comment (bakatzir) [inline Packs/Whois/ReleaseNotes/1_1_5.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

File: Packs/Whois/Integrations/Whois/Whois.py [hunk 9]
```diff
@@ -8037,7 +8037,7 @@ def parse_dates(dates):
                     hour = 0
                     minute = 0
                     second = 0
-                    print(e.message)  # FIXME: This should have proper logging of some sort...?
+                    demisto.debug(e.message)
```
Comment (bakatzir) [inline Packs/Whois/Integrations/Whois/Whois.py:8040]: nice @amshamah419 .

**Review by bakatzir — COMMENTED** [review]
Very nice.

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 10]
```diff
@@ -28,6 +28,7 @@ def handle_error(message, is_return_error):
         return_error(message)
     else:
         demisto.results(message)
+        sys.exit(0)
```
Comment (erezh31) [inline Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:31]: I think it's fine, @lperryp ?

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml (hunk 2 above)
Comment (lperryp) [reply Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml:128]: It's better to leave demisto/dl:1.3, the newer one is larger

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py (hunk 10 above)
Comment (lperryp) [reply Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:31]: @erezh31 I think so too (if it won't return an error )

**Comment by orhovy** [comment]
Opened a new pr due to too many conflicts

**Pull request closed without merging**